### PR TITLE
Set current language to layout

### PR DESCRIPTION
**Pull request by NeftaliYagua**
Set current language to layout

### CODE REVIEW

File: module/Application/view/layout/layout.phtml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php echo $this->doctype(); ?>
 
-<html lang="en">
+<html lang="<?php echo $this->plugin('translate')->getTranslator()->getLocale(); ?>">
```
Comment (Ocramius) [inline module/Application/view/layout/layout.phtml]: Needs escaping

**Comment by NeftaliYagua** [comment]
Ready!

File: module/Application/view/layout/layout.phtml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php echo $this->doctype(); ?>
 
-<html lang="en">
+<html lang="<?php echo $this->escapeHtml($this->plugin('translate')->getTranslator()->getLocale()); ?>">
```
Comment (Maks3w) [inline module/Application/view/layout/layout.phtml]: escapeHtmlAttr

**Comment by NeftaliYagua** [comment]
Ready!

**Comment by weierophinney** [comment]
We've removed translations from the skeleton for the upcoming v3 release; thank you for the effort, though!

**Pull request closed without merging**